src/Testing/CrawlerTrait.php

```diff
@@ -338,7 +338,7 @@ protected function seeJsonContains(array $data)
             json_decode($this->response->getContent(), true)
         ));
 
-        foreach (array_sort_recursive($data) as $key => $value) {
+        foreach (array_sort_recursive(json_decode(json_encode($data), true)) as $key => $value) {
```

Comment:
Not sure if you mean from a code style standpoint or implementation standpoint?  I'm just mimicking the implementation done earlier on in the method to the response data.

```
$actual = json_encode(array_sort_recursive(
    json_decode($this->response->getContent(), true)
));
```